dotnet6/cookiecutter-aws-sam-quick-start-sqs-dotnet/{{cookiecutter.project_name}}/template.yaml

```diff
@@ -0,0 +1,49 @@
+# This is the SAM template that represents the architecture of your serverless application
+# https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-template-basics.html
+
+# The AWSTemplateFormatVersion identifies the capabilities of the template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/format-version-structure.html
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  {{cookiecutter.project_name}}
+
+# Transform section specifies one or more macros that AWS CloudFormation uses to process your template
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/transform-section-structure.html
+Transform:
+- AWS::Serverless-2016-10-31
+
+# More info about Globals: https://github.com/awslabs/serverless-application-model/blob/master/docs/globals.rst
+Globals:
+  Function:
+    Timeout: 10
+
+# Resources declares the AWS resources that you want to include in the stack
+# https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/resources-section-structure.html
+Resources:
+  # This is an SQS queue with all default configuration properties. To learn more about the available options, see
+  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-sqs-queues.html
+  SimpleQueue:
+    Type: AWS::SQS::Queue
+  
+  # This is the Lambda function definition associated with the source code: sqs-payload-logger.js. For all available properties, see
+  # https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+  SQSPayloadLogger:
+    Type: AWS::Serverless::Function
+    Properties:
+      CodeUri: ./src/{{cookiecutter.project_name}}/
```

Comment:
@ammoheim and also here to add Graviton support 

[https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L55](https://github.com/aws/aws-sam-cli-app-templates/blob/6aaf6ac2e0ada5f87c4c6f69c5c1d8696513f2fe/dotnet6/cookiecutter-aws-sam-hello-step-functions-sample-app/%7B%7Bcookiecutter.project_name%7D%7D/template.yaml#L55)